torch/_dynamo/variables/functions.py

```diff
@@ -439,6 +443,20 @@ def call_method(
         elif name == "__iter__":
             # iter(gen) returns itself
             return self
+        elif name == "send":
+            # Sends a value into the generator function. Returns the next value
+            # yielded by the generator, or raises StopIteration if the generator
+            # exits without yielding another value
+            if self._is_generator_new() and len(args):
+                # can't send non-None value to a just-started generator
+                if not all(
+                    isinstance(arg, ConstantVariable) and arg.value is None
+                    for arg in args
+                ):
+                    raise_observed_exception(TypeError, tx)
```

Comment:
Can you write a test that exercises this (maybe with fullgraph=False)?

Reply:
There's one test that exercises this: `GeneratorCPythonTests.test_send_non_none_to_new_gen`